pkg/apis/extensions/v1alpha1/types_bastion.go

```diff
@@ -81,10 +81,10 @@ type BastionStatus struct {
 
 // BastionList is a collection of Bastions.
 type BastionList struct {
-	metav1.TypeMeta
+	metav1.TypeMeta `json:",inline"`
 	// Standard list object metadata.
-	metav1.ListMeta
+	metav1.ListMeta `json:"metadata"`
```

Comment:
Other places we use it as optional field.
https://github.com/gardener/gardener/blob/e1c28d64fbefbe23e5003d3f40966fc84d5f23c4/pkg/apis/extensions/v1alpha1/types_backupentry.go#L58

```suggestion
	// +optional
	metav1.ListMeta `json:"metadata,omitempty"`
```

Reply:
/hold
for clarification

Reply:
I pushed a2d44ff4580aeb8a530308128cfff75078d0afc3.